typescript-conditionals/main.ts

```diff
@@ -0,0 +1,140 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+
+function isUnderFive(x: number): boolean {
+  if (x < 5) {
+    return true;
+  } else {
+    return false;
+  }
+}
+const isUnderFiveResult: boolean = isUnderFive(3);
+console.log('value of isUnderFiveResult:', isUnderFiveResult);
+
+function isEvenNumber(x: number): boolean {
+  if (x % 2) {
+    return true;
+  } else {
+    return false;
+  }
+}
+const isEvenNumberResult: boolean = isEvenNumber(10);
+console.log('value of isEvenNumber:', isEvenNumberResult);
+
+const string1: string = 'Jennifer';
+
+function startsWithJ(string: string): boolean {
+  if (string.charAt(0) === 'J') {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+const startsWithJResult: boolean = startsWithJ(string1);
+console.log('value of startsWithJResult:', startsWithJResult);
+
+const person = {
+  name: 'Bryant',
+  age: 22,
+};
+
+function isOldEnoughToDrink(person: { name: string; age: number }): boolean {
+  if (person.age >= 21) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+const isOldEnoughToDrinkResult: boolean = isOldEnoughToDrink(person);
+console.log('value of is oldEnoughToDrink:', isOldEnoughToDrinkResult);
+
+function isOldEnoughToDrive(person: { name: string; age: number }): boolean {
+  if (person.age >= 16) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+const isOldEnoughToDriveResult: boolean = isOldEnoughToDrive(person);
+console.log('value of isOldEnoughToDrive:', isOldEnoughToDriveResult);
+
+function isOldEnoughToDrinkAndDrive(person: {
+  name: string;
+  age: number;
+}): boolean {
+  if (person.age >= 21) {
+    return false;
+  } else {
+    return false;
+  }
+}
```

Comment:
While this would be true under the assumption that you have to be older than 16 to be old than 21, you should still run a check for each item in the same condition.